site/_posts/2020-09-03-oidc-auth-api-usage.md

```diff
@@ -103,13 +103,13 @@ The bash example commands:
 # Accessing the ManageIQ API with the access_token from a JWT Token
 
    # Step 1: Request the JWT Token from the OpenID-Connect provider
-   jwt_token=$(curl -k -L --user ${user}:${password} -X POST -H "Content-Type: application/x-www-form-urlencoded" -d "grant_type=password" -d "client_id=${oidc_client_id}" -d "client_secret=${oidc_client_secret}" -d "username=${user}" -d "password=${password}" ${oidc_provider_token_endpoint} )
+   jwt_token=$(curl -k -L -X POST -H "Content-Type: application/x-www-form-urlencoded" -d "grant_type=password" -d "client_id=${oidc_client_id}" -d "client_secret=${oidc_client_secret}" -d "username=${user}" -d "password=${password}" -d "scope=openid" ${oidc_provider_token_endpoint} )
```

Comment:
I never needed to specify the scope for keycloak. do you need to use it for your setup ?

Reply:
Thanks for taking a look @abellotti.

It is not needed for Keycloak but it works OK with it for Keycloak. It is required for IAM.
So including it in the blog post ensures the example apply to both Keycloak and IAM.